HopsanCore/src/Quantities.cpp

```diff
@@ -56,13 +55,18 @@ hopsan::QuantityRegister::QuantityRegister()
 
     registerQuantity("Density", "kg/m^3");
 
-    registerQuantity("Frequency", "rad/s");
+    registerQuantity("Frequency", "Hz");
```

Comment:
Changing the base unit for a quantity may be dangerous, We need to verify what will happen to old models that have saved values in Frequency parameters. Will loading them still show and simulate the correct value.
For instance if it is loaded in HopsanGUI it may incorrectly re-scale the value to the new base unit. 

- [ ] figure out what may happen when old models are loaded when a quantity has changed base unit. It my be possible to auto update models as we have done in the passt. (needs implementation in both Core and GUI loader)

Reply:
Hz was not used as base unit before. So I guess that can't be mixed up (hopefully). However, the other way round you are right I guess. If a variable was added - e.g. with addInputVariable() - with quantity "Frequency" instead of unit "Hz" or "1/s", then it will be troublesome. Because Frequency was rad/s before and now is Hz in the commit. 
Just checked; there are actually a lot of components using "Frequency", for instance Hydraulic33Valve.hpp. I.e. these components uses rad/s and not Hz I guess. 
So yes, you are right, we have a problem there. Solution? 
- Two different quantites Frequency=rad/s and FrequencyHz=Hz?
- Update components that used Frequency to use AngularVelocity instead? Both were rad/s before. Then let Frequency be Hz.
- A ton of other alternatives? 

Hmm... It seams like components have a bit too much freedom to select quantities and units for the moment.

Reply:
> Two different quantites Frequency=rad/s and FrequencyHz=Hz?

In that case it should be called AngularFrequency and Frequency I think. But that would also be annoying, since you may want to convert unit from rad/s to Hz on the same signal.

Reply:
Isn't rad/s commonly used in control theory, while Hz is more common in signal processing? Then one or the other of these disciplines will be annoyed if we force everyone to use the same default unit.

Reply:
Hmm... You don't want to stand in between those camps I guess, so maybe we should avoid starting a war :-)